src/jshint.js

```diff
@@ -3413,88 +3409,31 @@ var JSHINT = (function() {
     });
   }
 
-  var conststatement = stmt("const", function(context) {
+  function variableStatement(type, statement, context) {
```

Comment:
sidenote, it also makes debugging a bit more painful (having to use conditional breakpoints is always annoying when it can be avoided)